www/src/data/sidebars/doc-links.yaml

```diff
@@ -101,6 +101,8 @@
               link: /docs/sourcing-from-private-apis/
             - title: Sourcing from (Headless) CMSs
               link: /docs/headless-cms/
+            - title: Sourcing Content from JSON or YAML
```

Comment:
This needs to move below the `items:` key so that the list of items shows up underneath the Headless CMS section and `Sourcing Content from JSON or YAML` stands alone 🙂 

```
- title: Sourcing from (Headless) CMSs
  link: /docs/headless-cms/
  items:
    # This list should only include the top 5 CMSs by Gatsby plugin usage
	...
- title: Sourcing Content from JSON or YAML
  link: /docs/sourcing-content-from-json-or-yaml/
```

sort of like this ☝️